src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SingleSpeedTalonDriveSubsystem.java

```diff
@@ -1,62 +1,40 @@
-    private WPI_TalonSRX m_rightMaster;
-    private WPI_TalonSRX m_leftMaster;
-    private WPI_TalonSRX[] m_rightSlaves;
-    private WPI_TalonSRX[] m_leftSlaves;
+    private TalonGroup m_rightTalonGroup;
+    private TalonGroup m_leftTalonGroup;
 
     public SingleSpeedTalonDriveSubsystem(
-        TalonIDGroup rightTalonIDs,
-        TalonIDGroup leftTalonIDs) {
-
-        // init right master
-        m_rightMaster = new WPI_TalonSRX(rightTalonIDs.getMasterTalonID());
-        m_rightMaster.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
-        m_rightMaster.setInverted(rightTalonIDs.getInvert());
-
-        // init left master
-        m_leftMaster = new WPI_TalonSRX(leftTalonIDs.getMasterTalonID());
-        m_leftMaster.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
-        m_leftMaster.setInverted(leftTalonIDs.getInvert());
-
-        // init right slaves
-        m_rightSlaves = new WPI_TalonSRX[rightTalonIDs.getSlaveTalonIDs().length];
-        for (int i = 0; i < m_rightSlaves.length; i++) {
-            m_rightSlaves[i] = new WPI_TalonSRX(rightTalonIDs.getSlaveTalonIDs()[i]);
-            m_rightSlaves[i].setInverted(rightTalonIDs.getInvert());
-            m_rightSlaves[i].follow(m_rightMaster);
-        }
-
-        // init left slaves
-        m_leftSlaves = new WPI_TalonSRX[leftTalonIDs.getSlaveTalonIDs().length];
-        for (int i = 0; i < m_leftSlaves.length; i++) {
-            m_leftSlaves[i] = new WPI_TalonSRX(leftTalonIDs.getSlaveTalonIDs()[i]);
-            m_leftSlaves[i].setInverted(leftTalonIDs.getInvert());
-            m_leftSlaves[i].follow(m_leftMaster);
-        }
-
-        // TODO: add more config to talons?
+        TalonGroup rightTalonGroup,
+        TalonGroup leftTalonGroup) {
+            m_rightTalonGroup = rightTalonGroup;
+            m_leftTalonGroup = leftTalonGroup;
     }
 
     @Override
     protected int getCurrentSpeedLeftClicksPerSecond() {
-        return m_leftMaster.getSelectedSensorPosition();
+        // could add a getSelectedSensorPosition command to the TalonGroup, so that
+        // we don't have to call getMasterTalon
+        return m_leftTalonGroup.getMasterTalon().getSelectedSensorPosition();
     }
 
     @Override
     protected int getCurrentSpeedRightClicksPerSecond() {
-        return m_rightMaster.getSelectedSensorPosition();
+        // could add a getSelectedSensorPosition command to the TalonGroup, so that
+        // we don't have to call getMasterTalon
+        return m_rightTalonGroup.getMasterTalon().getSelectedSensorPosition();
     }
 
     @Override
     protected void setProportional(double leftProportion, double rightProportion) {
-        m_leftMaster.set(ControlMode.PercentOutput, leftProportion);
-        m_rightMaster.set(ControlMode.PercentOutput, rightProportion);
+        // could add a set command to the TalonGroup, so that
+        // we don't have to call getMasterTalon
```

Comment:
I would recommend putting what you can in the TalonGroup, for sure.

Reply:
Oh whoops, I ended up going ahead and doing that, just forgot to remove the comment.